src/components/LinkChooser/utils/helpers.ts

```diff
@@ -0,0 +1,44 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
```

Comment:
maybe make the helpers a directory and provide multiple files, so you have a better context on the functionality

Reply:
done 👍